.github/workflows/publish-release-container.yml

```diff
@@ -0,0 +1,59 @@
+name: Build and Publish container for new releases
+
+on: [push] # debug
+  # release:
+  #   types:
+  #     - published
+
+jobs:
+  build:
+    name:                         Build and push image
+    runs-on:                      ubuntu-latest
+    env:
+      REGISTRY:                   docker.io/paritytech
```

Comment:
hmm, actually, don't we want to release this under `parity` and *not* `paritytech`? I think this is intended to be something that other people are going to be using.

Reply:
It was decided with @dvdplm that for the first time it will live under `paritytech`.

Reply:
I can be swayed either way though. Superficially it seems confusing to use two different collections of docker images, at least if they are labelled so confusingly similar. Sidecar **is** stable and it **is** an official product; it obviously does not have the same importance as polkadot.

Reply:
Well, the rule of thumb is that images under the `parity` namespace are generally to be for end-users, and `paritytech` is for in-house CI images, tooling, automation and stuff like that. For example, the telemetry backend and frontend are also both in the `parity` namespace. I would recommend we put this in `parity` too.

Reply:
Oh, I thought sidecar is still in beta. Then it totally makes sense to publish it as a product, in **parity** namespace.

Reply:
I'm not sure it really matters if it's in beta or not; the idea is that it's intended to be used externally, and I think changing the namespace down the road is a bad idea, because we have no idea who is going to use it where to notify them of the change.

Reply:
done

Reply:
> Oh, I thought sidecar is still in beta. Then it totally makes sense to publish it as a product, in **parity** namespace.

To give you an idea: sidecar has been used in production at exchanges for ~1 year now and some, like Huobi, has it as a pre-requisite for listing new tokens. It's legit. :)